genomic_data_service/region_indexer.py

```diff
@@ -274,9 +273,56 @@ def fetch_datasets(files, datasets):
+    
+    # get files in experiment TF ChIP-seq using assembly GRCh38
+    endpoint = "https://www.encodeproject.org/search/?type=Experiment&control_type!=*&status=released&assay_title=TF+ChIP-seq&assembly=GRCh38&field=files.accession&field=files.preferred_default&field=files.file_format&field=files.analyses.@id&field=default_analysis&format=json&limit=all"
+    experiments = requests.get(endpoint).json()["@graph"]
+    # get files in experiment DNase-seq using assembly GRCh38
+    endpoint = "https://www.encodeproject.org/search/?type=Experiment&control_type!=*&status=released&assay_title=DNase-seq&&assembly=GRCh38&field=files.accession&field=files.preferred_default&field=files.file_format&field=files.analyses.@id&field=default_analysis&format=json&limit=all"
+    experiments = experiments + requests.get(endpoint).json()["@graph"]
+    
+    # get files in footprints
+    endpoint = "https://www.encodeproject.org/search/?type=Annotation&annotation_type=footprints&assembly=GRCh38&field=files.accession&format=json&limit=all"
+    annotations = requests.get(endpoint).json()["@graph"]
+    # get files in PWMs
+    endpoint = "https://www.encodeproject.org/search/?type=Annotation&annotation_type=PWMs&assembly=GRCh38&field=files.accession&format=json&limit=all"
+    annotations = annotations + requests.get(endpoint).json()["@graph"]
+    # get files in eQTLs
+    endpoint = "https://www.encodeproject.org/search/?type=Annotation&annotation_type=eQTLs&assembly=GRCh38&field=files.accession&format=json&limit=all"
+    annotations = annotations + requests.get(endpoint).json()["@graph"]
+
+    for annotation in annotations:
+        files = annotation.get("files", [])
+        for file in files:
+            encode_accessions.append(file["accession"])
+    
+    for experiment in experiments:   
+        files = experiment.get("files", [])
+        default_analysis_id = experiment["default_analysis"]
+        for file in files:
+            if (
+                "preferred_default" in file
+                and file["preferred_default"] == True
+                and file["file_format"] == "bed"
+                and file["analyses"][0]["@id"] == default_analysis_id
+            ):
+                encode_accessions.append(file["accession"])
 
-def index_regulome_db(encode_accessions, local_files, filter_files=False):
+    # get files for chromatin state for grch38
+    endpoint = "https://www.encodeproject.org/search/?type=File&output_type=semi-automated+genome+annotation&status=released&assembly=GRCh38&lab.title=Manolis+Kellis%2C+Broad&file_format=bed&format=json&limit=all"
+    files = requests.get(endpoint).json()["@graph"]
+    for file in files:
+        encode_accessions.append(file["accession"])
+        
+    with open('encode_accessions_all.pickle', 'wb') as handle:
+        pickle.dump(encode_accessions, handle)
+    return encode_accessions
 
+def index_regulome_db(encode_accessions, local_files, filter_files=False):
```

Comment:
`encode_accessions` and `local_files` should probably have default value `None`?